StreamVideoSwiftUITests/Utils/ToastView_Tests.swift

```diff
@@ -21,7 +21,7 @@ final class ToastView_Tests: XCTestCase {
 
         
         // Then
-        assertSnapshot(matching: view, as: .image)
+        AssertSnapshot(view, perceptualPrecision: 0.96)
```

Comment:
nit: Probably we should move this value into a varialble (like defaultTimeout)

Reply:
good point, added